src/components/ebay-textbox/test/test.server.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable no-unused-expressions */
+
+const expect = require('chai').expect;
+const testUtils = require('../../../common/test-utils/server');
+const rootSelector = `span.textbox`;
+const inputSelector = `input.textbox__control`;
+const textareaSelector = `textarea.textbox__control`;
+
+test('renders default input textbox', context => {
+    const $ = testUtils.getCheerio(context.render());
+    expect($(rootSelector).length).to.eql(1);
+    expect($(inputSelector).length).to.eql(1);
+});
+
+test('renders input textbox with some value', context => {
+    const input = { value: 'this is texbox' };
+    const $ = testUtils.getCheerio(context.render(input));
+    expect($(inputSelector).val()).to.equal('this is texbox');
+});
+
+test('renders input textbox with a placeholder', context => {
```

Comment:
Remove, since this will be covered by `testHtmlAttributes` after the `marko.json` changes.